planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -478,9 +481,6 @@ class PlanningSceneMonitor : private boost::noncopyable
   // Callback for a new planning scene msg
   void newPlanningSceneCallback(const moveit_msgs::PlanningSceneConstPtr &scene);
 
-  // Called to update the planning scene with a new message.
-  void newPlanningSceneMessage(const moveit_msgs::PlanningScene& scene);
```

Comment:
just a guess, but wouldn't this break ABI too? I don't care about ABI, actually, but maybe we should go ahead and make it return bool while we're at it (doesn't break API).